File: js/ejer_19.js
salario=1300000
if (salario<1500000)
{
    salario=salario+(salario*0.0)
    console.log("Salario a pagar: " + salario)
}
else if (salario<3000000 )
{
    salario=salario+(salario*0.10)
    console.log("Salario a pagar: " + salario)
}
else if (salario>3000000)
{
    salario=salario+(salario*0.20)
    console.log("Salario a pagar: " + salario)  
}
else {
    console.log("Salario a pagar: " + salario)
}

File: python/ejer_19.py
salario=1300000
if (salario<1500000):
    salario_final=salario-(salario*0.0)
    print("Salario a pagar: ", salario_final)
elif (salario<=3000000):
    salario_final=salario-(salario*0.10)
    print("Salario a pagar: ", salario_final)
elif (salario>3000000):
    salario_final=salario-(salario*0.20)
    print("Salario a pagar: ", salario_final)
else:   
    print("Salario no valido")